packages/node-sdk/example/bucket.ts

```diff
@@ -0,0 +1,8 @@
+import { BucketClient } from "../src";
+
+export const bucket = new BucketClient({
+  secretKey: process.env.BUCKET_SECRET_KEY || "",
+});
+
+await bucket.initialize();
```

Comment:
Does this `await` actually work at module level?

Reply:
with modern Node.js it should work.